src/org/labkey/targetedms/TargetedMSManager.java

```diff
@@ -1451,6 +1458,25 @@ public static void purgeDeletedSampleFiles(long sampleFileId)
         // Delete from PeptideChromInfo
         execute("DELETE FROM " + getTableInfoGeneralMoleculeChromInfo() + " WHERE SampleFileId = ?", sampleFileId);
 
+        // Delete from QCTraceMetricValues
+        execute("DELETE FROM " + getTableQCTraceMetricValues() + " WHERE SampleFileId = ?", sampleFileId);
+
+        // Delete from QCEnabledMetrics
+        execute("DELETE FROM " + getTableInfoQCEnabledMetrics() + " WHERE metric IN ( " +
```

Comment:
Once these are detached from the SampleFileChromInfo, they should go back into TargetedMSListener

Reply:
done